robosat/tools/predict.py

```diff
@@ -34,47 +36,47 @@ def add_parser(subparser):
     parser.add_argument("--workers", type=int, default=0, help="number of workers pre-processing images")
     parser.add_argument("tiles", type=str, help="directory to read slippy map image tiles from")
     parser.add_argument("probs", type=str, help="directory to save slippy map probability masks to")
-    parser.add_argument("--model", type=str, required=True, help="path to model configuration file")
     parser.add_argument("--dataset", type=str, required=True, help="path to dataset configuration file")
+    parser.add_argument("--masks_output", action="store_true", help="output masks rather than probs")
+    parser.add_argument("--leaflet", type=str, help="leaflet client base url")
 
     parser.set_defaults(func=main)
 
 
 def main(args):
-    model = load_config(args.model)
     dataset = load_config(args.dataset)
+    num_classes = len(dataset["common"]["classes"])
 
-    cuda = model["common"]["cuda"]
-
-    device = torch.device("cuda" if cuda else "cpu")
+    if torch.cuda.is_available():
+        device = torch.device("cuda")
+        torch.backends.cudnn.benchmark = True
+    else:
+        device = torch.device("cpu")
```

Comment:
:+1: I'm fine with switching the logic here to always using cuda if available. I think users still have the cuda environment variables they can set if they really don't want to use gpus but they do have gpus available.